docs/docker_quickstart.md

```diff
@@ -68,20 +67,19 @@ Create a new directory and place the [docker-compose file](https://github.com/fr
 The above snippet creates a new directory called `ft_userdata`, downloads the latest compose file and pulls the freqtrade image.
 The last 2 steps in the snippet create the directory with `user_data`, as well as (interactively) the default configuration based on your selections.
 
-!!! Question "How to edit the bot configuration?"
-    You can edit the configuration at any time, which is available as `user_data/config.json` (within the directory `ft_userdata`) when using the above configuration.
-
-    You can also change the both Strategy and commands by editing the command section of your `docker-compose.yml` file.
+### Question: "How to edit the bot configuration?"
+You can edit the configuration at any time, which is available as `user_data/config.json` (within the directory `ft_userdata`) when using the above configuration.
+You can also change the both Strategy and commands by editing the command section of your `docker-compose.yml` file.
 
-#### Adding a custom strategy
+##### Adding a custom strategy
 
 1. The configuration is now available as `user_data/config.json`
 2. Copy a custom strategy to the directory `user_data/strategies/`
 3. Add the Strategy' class name to the `docker-compose.yml` file
 
 The `SampleStrategy` is run by default.
 
-!!! Warning "`SampleStrategy` is just a demo!"
+#### Warning "`SampleStrategy` is just a demo!"
```

Comment:
the `!!!` have been here intentional - as these render the "Warning" or "note" boxes on freqtrade.io.